app/email_batching.go

```diff
@@ -239,59 +269,31 @@ func (es *EmailService) sendBatchedEmailNotification(userID string, notification
-	var templateName = "post_batched_post_generic"
-	if emailNotificationContentsType == model.EMAIL_NOTIFICATION_CONTENTS_FULL {
-		templateName = "post_batched_post_full"
+	firstSender, err := es.srv.Store.User().Get(context.Background(), notifications[0].post.UserId)
+	if err != nil {
+		mlog.Warn("Unable to find sender of post for batched email notification")
 	}
 
-	data := es.newEmailTemplateData(userLocale)
-	data.Props["Button"] = translateFunc("api.email_batching.render_batched_post.go_to_post")
-	data.Props["PostMessage"] = es.srv.GetMessageForNotification(notification.post, translateFunc)
-	data.Props["PostLink"] = siteURL + "/" + notification.teamName + "/pl/" + notification.post.Id
-	data.Props["SenderName"] = sender.GetDisplayName(displayNameFormat)
-
-	tm := time.Unix(notification.post.CreateAt/1000, 0)
-	timezone, _ := tm.Zone()
-
-	data.Props["Date"] = translateFunc("api.email_batching.render_batched_post.date", map[string]interface{}{
-		"Year":     tm.Year(),
-		"Month":    translateFunc(tm.Month().String()),
-		"Day":      tm.Day(),
-		"Hour":     tm.Hour(),
-		"Minute":   fmt.Sprintf("%02d", tm.Minute()),
-		"Timezone": timezone,
+	data := es.newEmailTemplateData(user.Locale)
+	data.Props["SiteURL"] = siteURL
+	data.Props["Title"] = translateFunc("api.email_batching.send_batched_email_notification.title", len(notifications)-1, map[string]interface{}{
+		"SenderName": firstSender.GetDisplayName(displayNameFormat),
 	})
-
-	if channel.Type == model.CHANNEL_DIRECT {
-		data.Props["ChannelName"] = translateFunc("api.email_batching.render_batched_post.direct_message")
-	} else if channel.Type == model.CHANNEL_GROUP {
-		data.Props["ChannelName"] = translateFunc("api.email_batching.render_batched_post.group_message")
-	} else {
-		// don't include channel name if email notification contents type is set to generic
-		if emailNotificationContentsType == model.EMAIL_NOTIFICATION_CONTENTS_FULL {
-			data.Props["ChannelName"] = channel.DisplayName
-		} else {
-			data.Props["ChannelName"] = translateFunc("api.email_batching.render_batched_post.notification")
-		}
+	data.Props["SubTitle"] = translateFunc("api.email_batching.send_batched_email_notification.subTitle")
+	data.Props["Button"] = translateFunc("api.email_batching.send_batched_email_notification.button")
+	data.Props["ButtonURL"] = siteURL
+	data.Props["Posts"] = postsData
+	data.Props["MessageButton"] = translateFunc("api.email_batching.send_batched_email_notification.messageButton")
+	data.Props["NotificationFooterTitle"] = translateFunc("app.notification.footer.title")
+	data.Props["NotificationFooterInfoLogin"] = translateFunc("app.notification.footer.infoLogin")
+	data.Props["NotificationFooterInfo"] = translateFunc("app.notification.footer.info")
+
+	renderedPage, renderErr := es.srv.TemplatesContainer().RenderToString("messages_notification", data)
+	if renderErr != nil {
+		mlog.Error("Unable to render email", mlog.Err(renderErr))
```

Comment:
Gentle reminder to check our logging [principles](https://developers.mattermost.com/contribute/server/style-guide/#log-levels) before merging this change.